packages/calling/src/CallingClient/registration/register.ts

```diff
@@ -681,13 +689,17 @@ export class Registration implements IRegistration {
   private startKeepaliveTimer(url: string, interval: number) {
     let keepAliveRetryCount = 0;
     this.clearKeepaliveTimer();
+    const isCCFlow = this.serviceData.indicator === ServiceIndicator.CONTACT_CENTER;
```

Comment:
Cant we use the above isCCFlow variable here also?

Reply:
Maybe we can define a class variable isCCFlow, because if callingClient is created with cc service indicator it will always remain a cc flow.

Reply:
Done. Made `isCCFlow` a private class level variable and updated the logic everywhere.